src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -555,61 +557,111 @@ const ariaLabels = computed(() => ({
+			:select-text="t('Pick')"
+			six-weeks="fair"
+			:teleport="appendToBody ? (target || undefined) : false"
+			text-input
+			:week-num-name
+			:week-numbers="showWeekNumber ? { type: 'iso' } : undefined"
+			:week-start
+			v-bind="pickerType"
+			@update:model-value="onUpdateModelValue">
+			<template #input-icon>
+				<NcIconSvgWrapper :path="mdiCalendarBlank" :size="20" />
+			</template>
+			<template #clear-icon>
+				<NcIconSvgWrapper inline :path="mdiClose" :size="20" />
+			</template>
+			<template #clock-icon>
+				<NcIconSvgWrapper inline :path="mdiClock" :size="20" />
+			</template>
+			<template #arrow-left>
+				<NcIconSvgWrapper inline :path="mdiChevronLeft" :size="20" />
+			</template>
+			<template #arrow-right>
+				<NcIconSvgWrapper inline :path="mdiChevronRight" :size="20" />
+			</template>
+			<template #arrow-down>
+				<NcIconSvgWrapper inline :path="mdiChevronDown" :size="20" />
+			</template>
+			<template #arrow-up>
+				<NcIconSvgWrapper inline :path="mdiChevronUp" :size="20" />
+			</template>
+			<template v-if="showTimezoneSelect" #action-extra>
+				<NcTimezonePicker v-model="timezoneId"
+					class="vue-date-time-picker__timezone"
+					:append-to-body="false"
+					:input-label="t('Timezone')" />
+			</template>
+		</VueDatePicker>
+		<Teleport to="body" :disabled="!appendToBody">
+			<div ref="target" class="vue-date-time-picker-wrapper" />
+		</Teleport>
+	</span>
 </template>
 
 <style scoped lang="scss">
-@import '@vuepic/vue-datepicker/dist/main.css';
+@use "sass:meta";
+
+.vue-date-time-picker-wrapper {
+	// This is under :root in @vuepic/vue-datepicker/dist/main.css, so importing it scoped won't work
+	--dp-common-transition: all 0.1s ease-in;
+	--dp-menu-padding: 6px 8px;
+	--dp-animation-duration: 0.1s;
+	--dp-menu-appear-transition-timing: cubic-bezier(.4, 0, 1, 1);
+	--dp-transition-timing: ease-out;
+	--dp-action-row-transtion: all 0.2s ease-in;
+	--dp-font-family: -apple-system, blinkmacsystemfont, "Segoe UI", roboto, oxygen, ubuntu, cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+	// --dp-border-radius: 4px;
+	--dp-cell-border-radius: 4px;
+	--dp-transition-length: 22px;
+	--dp-transition-timing-general: 0.1s;
+	--dp-button-height: 35px;
+	--dp-month-year-row-height: 35px;
+	--dp-month-year-row-button-size: 25px;
+	--dp-button-icon-height: 20px;
+	--dp-calendar-wrap-padding: 0 5px;
+	--dp-cell-size: 35px;
+	--dp-cell-padding: 5px;
+	--dp-common-padding: 10px;
+	// --dp-input-icon-padding: 35px;
+	--dp-input-padding: 6px 30px 6px 12px;
+	--dp-menu-min-width: 260px;
+	--dp-action-buttons-padding: 1px 6px;
+	--dp-row-margin: 5px 0;
+	--dp-calendar-header-cell-padding: 0.5rem;
+	--dp-multi-calendars-spacing: 10px;
+	--dp-overlay-col-padding: 3px;
+	--dp-time-inc-dec-button-size: 32px;
+	--dp-font-size: 1rem;
+	--dp-preview-font-size: 0.8rem;
+	--dp-time-font-size: 2rem;
+	--dp-action-button-height: 22px;
+	--dp-action-row-padding: 8px;
+	--dp-direction: ltr;
```

Comment:
TBH I do not like this as this is error prone if there are new styles added (non breaking library change).
But I see that there is little we can do about this.

Only alternative I see would be patching the imported CSS to replace the `:root` selector.

Reply:
Did not find anything better maintained as this.